store/sqlstore/group_store.go

```diff
@@ -1198,6 +1198,12 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 		`, opts.NotAssociatedToChannel)
 	}
 
+	if opts.Since > 0 {
+		groupsQuery = groupsQuery.Where(sq.Gt(map[string]interface{}{
+			"g.UpdateAt": opts.Since,
+		}))
```

Comment:
I think we will also either need to include deleted groups by default when passing `Since` or add a new option to `IncludeDeleted` since we need a way for the client to know which groups to remove from the state

Reply:
Updated to return deleted groups with "since". Since it will only return those deleted "since", not all deleted groups, it made sense to do this by default.